src/main/java/domain/piece/Piece.java

```diff
@@ -19,25 +17,21 @@ public boolean isEqualColor(final Color target) {
         return this.color == target;
     }
 
+    public boolean isNotEqualColor(final Color target) {
+        return this.color != target;}
```

Comment:
```suggestion
        return this.color != target;
    }
```
또한 파라미터는 target인데 Color가 오고 있습니다! 외부에서 getter를 사용하고 있겠네요 👀

Reply:
기존 isEqualColor의 경우 외부에서 getter를 사용하고 있어, Piece를 파라미터로 받아 비교하도록 변경했습니다.
isNotEqualColor의 경우 사용자 턴을 위해 사용된 메서드로, Color를 파라미터로 받아도록 변경했습니다. 
[궁금한점]
두 메서드명을 비교하면, 같은 타입의 파라미터를 받을 것처럼 예상됩니다. 
메서드명을 수정하는것이 나을까요?

Reply:
넵 말씀하신대로 오해의 여지가 있을 수 있겠네요 👀
메서드 오버로딩으로 볼 수도 있고 컴파일 타임에 바로 알 수 있기 때문에 
메서드로 적절히 표현할 수 있다면 변경하고 아니라면 현재도 괜찮아보입니다 😊